RecoTracker/Configuration/python/RecoTracker_EventContent_cff.py

```diff
@@ -2,87 +2,53 @@
 
 import FWCore.ParameterSet.Config as cms
 
-#Full Event content 
-RecoTrackerFEVT = cms.PSet(
+#AOD content
+RecoTrackerAOD = cms.PSet(
     outputCommands = cms.untracked.vstring(
         'keep recoTracks_generalTracks_*_*', 
-        'keep recoTrackExtras_generalTracks_*_*',
-        'keep TrackingRecHitsOwned_generalTracks_*_*',
-        'keep *_generalTracks_MVAValues_*',
-        'keep TrackingRecHitsOwned_extraFromSeeds_*_*',
-        'keep uints_extraFromSeeds_*_*',                                   
-        'keep recoTracks_beamhaloTracks_*_*', 
-        'keep recoTrackExtras_beamhaloTracks_*_*', 
-        'keep TrackingRecHitsOwned_beamhaloTracks_*_*',
-        'keep recoTracks_conversionStepTracks_*_*', 
-        'keep recoTrackExtras_conversionStepTracks_*_*', 
-        'keep TrackingRecHitsOwned_conversionStepTracks_*_*',
-        'keep *_ctfPixelLess_*_*', 
-        'keep *_dedxTruncated40_*_*',
-        'keep *_dedxHitInfo_*_*',
+        'keep recoTracks_conversionStepTracks_*_*',
+        'keep recoTracks_beamhaloTracks_*_*',
+        'keep recoTracks_ctfPixelLess_*_*', 
         'keep *_dedxHarmonic2_*_*',
         'keep *_dedxPixelHarmonic2_*_*',
+        'keep *_dedxHitInfo_*_*',
         'keep *_trackExtrapolator_*_*',
-        'keep recoTracks_cosmicDCTracks_*_*',
-        'keep recoTrackExtras_cosmicDCTracks_*_*',
-        'keep TrackingRecHitsOwned_cosmicDCTracks_*_*',
-     )
+        'keep *_generalTracks_MVAValues_*',
+        'keep *_generalTracks_MVAVals_*'
```

Comment:
Just noticing that before this PR "*_generalTracks_MVAVals_*" was not kept in the FEVT format, while being there in the RECO/AOD ones: this PR also fixes that inconsistency